modules/dkan/dkan_datastore/includes/Datastore.inc

```diff
@@ -40,7 +40,14 @@ abstract class Datastore {
     if (!isset($node[$uuid])) {
       $nid = $this->getNid($uuid);
       if (!$nid) {
-        throw new Exception(t("uuid !uuid not found.", array('!uuid' => $uuid)));
+        try {
+          if (!$nid) {
+            throw new Exception(t("uuid !uuid not found.", array('!uuid' => $uuid)));
```

Comment:
@janette If we are throwing an exception and immediately catching it, I think we can remove the exception and simply add the watchdog_statement. Thoughts?